ci-operator/step-registry/osde2e/test/osde2e-test-commands.sh

```diff
@@ -21,7 +21,15 @@ then
    export TEST_KUBECONFIG="${SHARED_DIR}/kubeconfig"
 fi
 
+echo "Extracting CLUSTER_ID..."
```

Comment:
Why `echo` this?  None of the other `exports` are logging anything.  Let's maintain this consistency.